tests/test_transformer/test_generic_transformer.py

```diff
@@ -15,6 +17,69 @@
 # pylint: disable=missing-function-docstring, missing-class-docstring
 
 
+def test_functions_transformer_in_pipeline(X):
+    pipeline = make_pipeline(
+        FunctionsTransformer([("a", np.log1p, None), ("b", np.sqrt, None)])
+    )
+    result = pipeline.fit_transform(X)
+    expected_a = np.array(
+        [
+            0.6931471805599453,
+            1.0986122886681098,
+            1.3862943611198906,
+            1.6094379124341003,
+            1.791759469228055,
+            1.9459101490553132,
+            2.0794415416798357,
+            2.1972245773362196,
+            2.302585092994046,
+            2.3978952727983707,
+        ]
+    )
+
+    expected_b = np.array(
+        [
+            1.0488088481701516,
+            1.4832396974191326,
+            1.816590212458495,
+            2.0976176963403033,
+            2.345207879911715,
+            2.569046515733026,
+            2.7748873851023217,
+            2.9664793948382653,
+            3.146426544510455,
+            3.1780497164141406,
+        ]
+    )
+
+    assert np.array_equal(result["a"].values.round(6), expected_a.round(6))
+    assert np.array_equal(result["b"].values.round(6), expected_b.round(6))
+    assert pipeline.steps[0][0] == "functionstransformer"
+
+
+def test_functions_transformer_raises_error(X) -> None:
```

Comment:
Is this redundant, since you now have `check_ready_to_transform()`?

Reply:
This is actually a good point. Since the other function is also tested you are probably right, but I would like to leave it in. More tests don't break the project.